README.md

```diff
@@ -188,12 +177,42 @@ spec:
     argument: "0.25"
   java:
     image: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java:latest # <1>
-EOF
+  nodejs:
+    image: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:latest
+  EOF
 ```
 
+The above CR can be queried by `kubectl get otelinst`.
+
 1. Container image with [OpenTelemetry Java auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation). The image must contain the Java agent JAR `/javaagent.jar`, and the operator will copy it to a shared volume mounted to the application container.
 
-The above CR can be queried by `kubectl get otelinst`.
+#### Java
+
+The operator can inject OpenTelemetry [Java auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation).
+The injection of the Java agent can be enabled by adding an annotation to the namespace, so that all pods within that namespace will get the instrumentation, or by adding the annotation to individual PodSpec objects, available as part of Deployment, Statefulset, and other resources.
+
+```bash
+instrumentation.opentelemetry.io/inject-java: "true"
+```
+
+The value can be 
+* `"false"` - do not inject
+* `"true"` - inject and `Instrumentation` resource from the namespace.
+* `"my-instrumentation"` - name of `Instrumentation` CR instance.
+
+#### NodeJS
+
+The operator can inject OpenTelemetry [NodeJS auto-instrumentation](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node).
+The injection of the NodeJS instrumentation can be enabled by adding an annotation to the namespace, so that all pods within that namespace will get the instrumentation, or by adding the annotation to individual PodSpec objects, available as part of Deployment, Statefulset, and other resources.
+
+```bash
+instrumentation.opentelemetry.io/inject-nodejs: "true"
+```
+
+The value can be
```

Comment:
I think we should DRY this, the semantics for the annotation are the same for all languages, no need to repeat this.